lib/fluent/plugin_helper/child_process.rb

```diff
@@ -281,7 +282,7 @@ def child_process_execute_once(
           stderrio.set_encoding(external_encoding, internal_encoding, encoding_options)
           stderrio_in_use = true
         else
-          stderrio.reopen(IO::NULL) if stderrio && stderrio == :discard
```

Comment:
Just checking, This was a bug, right? `stderrio` cannot be a `:discard`.

Reply:
Yes. This is typo.